middlewares/validators/newsletter.js

```diff
@@ -0,0 +1,19 @@
+const joi = require("joi");
+
+const validateEmail = async (req, res, next) => {
+  const schema = joi.object().strict().keys({
+    email: joi.string().required(),
```

Comment:
Could we look into validation of email via joi....
Since this is particularly on newsletter service via email
I feel We can add extra validation checks for emails

Reply:
good point, will check validation for email.

Reply:
added @harshith-venkatesh , pls have a look

Reply:
Can't we use email validator of joi? 

Ref - https://joi.dev/api/?v=17.6.0

Reply:
thanks, done